odoo/tests/common.py

```diff
@@ -537,8 +537,20 @@ def executable(self):
 
         raise unittest.SkipTest("Chrome executable not found")
 
+    def _spawn_chrome(self):
+        pid = os.fork()
+        if pid != 0:
+            return pid
+        else:
+            if os.name == 'posix':
+                # prevent chrome crash because of huge memory reservation for V8
+                rlimit = resource.RLIMIT_RSS if platform.system() == 'Darwin' else resource.RLIMIT_AS
```

Comment:
Can skip everything for darwin: issue is when we're rlimiting vmem because pointer compression feature performs a large reservation upfront, but we're not rlimiting vmem on OSX because the OS almost always does a huge reservation upfront for all processes anyway, used to be processes got ~2.4GB VM by default and I think it's higher now (it's not rare to have aggregate VM sizes in the 200~300GB ranges). So the issue should not occur there.

Reply:
I meant explicit Python comments, in addition to the commit message that should have the same info